main.qml

```diff
@@ -42,7 +42,7 @@ import "wizard"
 
 ApplicationWindow {
     id: appWindow
-
+    title: "Monero GUI"
```

Comment:
Not saying this is a bad title, but I wonder if "Monero Wallet" isn't a better title? The GUI part feels redundant since in a desktop environment everyting that is shown is GUI:s. And it is a wallet (even if it has some daemon controlling capabilities, that is just some bonus features and not the primary function), hence my suggestion "Monero Wallet".